### PR TITLE
docs: add instructions for using clip server hosted by jina

### PR DESCRIPTION
**Pull request by ZiniuYu**

### CODE REVIEW

File: docs/hosting/by-jina.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+# Hosted by Jina AI
```
Comment (numb3r3) [inline docs/hosting/by-jina.md:1]: What's the difference betwee *hostted by Jina AI* and *hosted by jCloud*? I think this title is misleading. Maybe we can just use **# Cloud Hosting**?

Comment (numb3r3) [reply docs/hosting/by-jina.md:1]: I think this page would explain the SaaS product.

**Review by numb3r3 — CHANGES_REQUESTED** [review]

File: docs/hosting/by-jina.md (hunk 1 above)
Comment (ZiniuYu) [reply docs/hosting/by-jina.md:1]: One is hosted **by** Jina, right now the user can only access it but not control it.
The other is hosted **on** Jcloud, the user has full control of it

Comment (numb3r3) [reply docs/hosting/by-jina.md:1]: But this title does not tell the difference

**Review by numb3r3 — APPROVED** [review]
LGTM

**Pull request merged**